app/routes.py

```diff
@@ -1,2 +1,61 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
 
+planet_bp = Blueprint("planets", __name__, url_prefix= "/planets")
+
+@planet_bp.route("", methods=["GET"])
+def get_all_planets():
+    all_planets = []
+    if len(request.args) != 0:
+        planets = Planet.query.filter_by(**request.args)
+    else:
+        planets = Planet.query.all()
+
+    for planet in planets:
+        all_planets.append(planet.to_json())
+
+    return jsonify(all_planets)
+
+@planet_bp.route("/<planet_id>", methods=["GET"])
+def get_one_planet(planet_id):
+    one_planet = Planet.validate_id(planet_id)
+
+    return one_planet.to_json()
+
+@planet_bp.route("", methods=["POST"])
+def create_planet():
+    planet_data = request.get_json()
+    new_planet = Planet(
+        name=planet_data['name'],
+        description=planet_data['description'],
+        has_life=planet_data['has_life']
+        )
+    
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return jsonify(f"Planet {new_planet.name} successfully created"), 201
+
+@planet_bp.route("/<planet_id>", methods=["PUT"])
+def update_planet(planet_id):
+    planet = Planet.validate_id(planet_id)
+
+    new_data = request.get_json()
+
+    planet.name = new_data['name']
+    planet.description = new_data['description']
+    planet.has_life = new_data['has_life']
```

Comment:
This could be pulled into an instance method of Planet to keep your route short and sweet. Here's an [example](https://github.com/ada-c17/flasky/commit/6d43a9fe543bab8d5a8ec94484267899be483063#diff-7d18e19033b5c348d84d50ef5c7e523b123c34305df76fb7eff28b7fafe09beeR21-R26) of that in Cat for Flasky

Would be good to work in some validation in here too so that the client gets an error message in the response with meaningful feedback if they don't provide all the fields to update